src/lib/src/items-list.ts

```diff
@@ -45,6 +45,13 @@ export class ItemsList {
         this._selected = [];
     }
 
+    unSelectLastItem() {
+        if (this._selected.length) {
+            this._selected[this._selected.length - 1].selected = false;
+            this._selected.splice(this._selected.length - 1, 1);
+        }
+    }
```

Comment:
I prefer to always return as fast as possible, then code is more readable, see `markSelection` as an example.
```
if (this._selected.length === 0) {
   return;
}

// ok it is safe to do logic here
```